slack_reporter_config.example.py

```diff
@@ -0,0 +1,4 @@
+auth_token = "xoxb-secret-goes-here" # Should start with "xoxb-"
+board_name = "example board"
+
+channel_id = "C12345ABCDE"
```

Comment:
This is the channel ID, _not_ the channel name! It can be found by clicking on a channel title and scrolling to the bottom of the properties.